lib/twingly/url.rb

```diff
@@ -23,13 +23,22 @@ class URL
       PublicSuffix::DomainInvalid,
     ].freeze
     NBSP = "\u00A0"
+    SPACE = "\u0020"
+    WHITESPACE_CHARS = [
+      NBSP,
+      SPACE,
+    ].join.freeze
+    WHITESPACE = /\A[#{WHITESPACE_CHARS}]+|[#{WHITESPACE_CHARS}]+\z/.freeze
```

Comment:
`LEADING_AND_TRAILING_WHITESPACE`... short and concise :D

Reply:
ae9e161